### PR TITLE
Adds survey app

### LINKED ISSUE #104: Add helpers to manage Survey app
Following the new `BaseTemplate` helpers, we should add support for the Survey app

### PR DESCRIPTION
**Pull request by ajsantander**
:checkered_flag: *Ready for Review* :checkered_flag: 
Fix #104

### CODE REVIEW

File: templates/bare/package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "@aragon/apps-vault": "^4.1.0",
     "@aragon/apps-voting": "^2.1.0",
     "@aragon/apps-finance": "^3.0.0",
+    "@aragon/apps-survey": "^1.0.0",
```
Comment (sohkai) [inline templates/bare/package.json]: I don't think these other templates need the survey dependency?

File: shared/contracts/BaseTemplate.sol [hunk 2]
```diff
@@ -179,6 +182,20 @@ contract BaseTemplate is APMNamehash, IsContract {
         _removePermissionFromTemplate(_acl, _tokenManager, _tokenManager.MINT_ROLE());
     }
 
+    /* SURVEY */
+
+    function _installSurveyApp(Kernel _dao, MiniMeToken _token, uint64 _minParticipationPct, uint64 _surveyTime) internal returns (Survey) {
+        Survey survey = Survey(_installNonDefaultApp(_dao, SURVEY_APP_ID));
+        survey.initialize(_token, _minParticipationPct, _surveyTime);
```
Comment (sohkai) [inline shared/contracts/BaseTemplate.sol]: Once #125 is merged, we should use the optimized version too :).

File: templates/bare/package.json (hunk 1 above)
Comment (facuspagnuolo) [reply templates/bare/package.json]: I used to have problems with this as well, for some reason the templates using the `BaseTemplate` were asking for all the apps being used by the base :/
Will merge and polish it if possible

File: shared/contracts/BaseTemplate.sol (hunk 2 above)
Comment (facuspagnuolo) [reply shared/contracts/BaseTemplate.sol]: Merging both.. will provide a third PR to update this :)

File: templates/bare/package.json (hunk 1 above)
Comment (ajsantander) [reply templates/bare/package.json]: I was experiencing the problem of the templates not compiling if the Survey app was not added. Same thing happened when I was adding the Payroll app.

**Pull request merged**